### PR TITLE
feat: move kro-based validator system to protocol

### PR DESCRIPTION
**Pull request by 0xHansLee**

### CODE REVIEW

File: specs/protocol/validator-v1/validator-pool.md
Comment (0xHansLee) [inline specs/protocol/validator-v1/validator-pool.md:1]: This is just a moved out from the previous `validator.md` file

File: specs/protocol/validator.md [hunk 1]
```diff
@@ -43,21 +38,25 @@ with a bond at stake if the proof is wrong.
...
-The validator's role is to construct and submit output roots, which are commitments to the L2's state,
-to the `L2OutputOracle` contract on L1. To do this, the validator periodically
-queries the [rollup node](rollup-node.md) for the latest output root derived from the latest
-[finalized][finality] L1 block. It then takes the output root and
-submits it to the `L2OutputOracle` contract on L1.
+> **_NOTE:_**
+> Before the introduction of the KRO governance token, the
+> [validator system was ETH-based](./validator-v1/validator-pool.md). With introduction of KRO token, the
+> [KRO-based validator system](./validator-v2/overview.md) has been introduced, mitigating several limitations of the
+> ETH-based system.
```
Comment (0xHansLee) [inline specs/protocol/validator.md]: This is added.

File: specs/protocol/validator.md
Comment (seolaoh) [inline specs/protocol/validator.md:1]: How about renaming this one to `validation.md`?

File: specs/protocol/validator.md [hunk 2]
```diff
@@ -43,21 +38,25 @@ with a bond at stake if the proof is wrong.
...
 
-The validator's role is to construct and submit output roots, which are commitments to the L2's state,
-to the `L2OutputOracle` contract on L1. To do this, the validator periodically
-queries the [rollup node](rollup-node.md) for the latest output root derived from the latest
-[finalized][finality] L1 block. It then takes the output root and
-submits it to the `L2OutputOracle` contract on L1.
+> **_NOTE:_**
+> Before the introduction of the KRO governance token, the
+> [validator system was ETH-based](./validator-v1/validator-pool.md). With introduction of KRO token, the
+> [KRO-based validator system](./validator-v2/overview.md) has been introduced, mitigating several limitations of the
```
Comment (seolaoh) [inline specs/protocol/validator.md]:
```suggestion
> system has been transitted to [KRO-based validator system](./validator-v2/overview.md), mitigating several limitations of the
```

File: specs/protocol/validator.md
Comment (seolaoh) [inline specs/protocol/validator.md:1]: Please remove `REQUIRED_BOND_AMOUNT` and `REWARD_DIVIDER` in Constants section.

File: specs/protocol/validator-v1/validator-pool.md
Comment (seolaoh) [inline specs/protocol/validator-v1/validator-pool.md:1]: Please add Constants section.

File: specs/glossary.md [hunk 3]
```diff
@@ -898,20 +900,20 @@ An L1 contract to which [L2 output roots][l2-output] are posted by the [validato
...
 
 An [L1] contract that determines [validator] eligibility, selects the [validator] of next round, and manages bonding for
-[L2 output roots][l2-output] submissions in [Validator System V1][validator-system-v1].
+[L2 output roots][l2-output] submissions in [ETH-based Validator System][eth-based-validator-system].
 
 ## Validator Manager Contract
 
 [validator-manager-contract]: glossary.md#validator-manager-contract
 
 An [L1] contract that manages the set of [validators], selects the validator of next [priority round][priority-round],
```
Comment (seolaoh) [inline specs/glossary.md]:
```suggestion
An [L1] contract that manages the set of [validators][validator], selects the validator of next [priority round][priority-round],
```

File: specs/protocol/validator.md
Comment (0xHansLee) [reply specs/protocol/validator.md:1]: It looks good.

**Review by seolaoh — APPROVED** [review]
LGTM

**Pull request merged**